ci/axis/dask.yaml

```diff
@@ -25,9 +27,17 @@ UCX_PY_VER:
 
 excludes:
   - DOCKER_FILE: dask.Dockerfile
-    BUILD_IMAGE: gpuci/distributed
+    BUILD_IMAGE:
```

Comment:
I"m exactly sure how this should go but I would've thought we wanted the list break out like:

```
  - DOCKER_FILE: dask.Dockerfile
    BUILD_IMAGE: gpuci/distributed
  - DOCKER_FILE: distributed.Dockerfile
    BUILD_IMAGE: gpuci/dask
  - DOCKER_FILE: dask_sql.Dockerfile
    BUILD_IMAGE: gpuci/dask_sql
```

Reply:
This list is specifically for what builds we want excluded from the matrix - for example, we would want `dask_sql.Dockerfile` to *only* build and publish to `gpuci/dask_sql`, so we would exclude `BUILD_IMAGE=gpuci/dask` or `gpuci/distributed` so that it doesn't build and publish there too.